src/test/file.rs

```diff
@@ -1442,53 +1462,59 @@ impl DateTimeSpecification {
         format!("{}", time.format(&format))
     }
 
+    pub fn apply_modifier(
```

Comment:
Looks like this method is on the instance of the struct (&self) but we don't use self anywhere and instead pass in the variables. I'd either rely on &self and leverage the fields, or remove &self so it's "static" on the type.